content/docs/setup/kubernetes/quick-start-gke-dm/index.md

```diff
@@ -166,17 +168,17 @@ If you are using Cloud Shell rather than the installed `gcloud` client, you can
 
 ### Grafana
 
-Set up a tunnel to Grafana:
+1.  Set up a tunnel to Grafana:
```

Comment:
here and below where an ordered list is used please don't add an extra space.  (e.g. this should do `1. Set up a tunnel to Grafana:`